From d15be88ccb1b7df9a0b6db2967131521b526918b Mon Sep 17 00:00:00 2001
From: Mark Stemm <mark.stemm@gmail.com>
Date: Tue, 15 Aug 2017 17:49:53 -0700
Subject: [PATCH] Don't include curlbuild.h

It's not required to build and it's not present with some newer versions
of libcurl, so removing it.

Thanks to https://github.com/hhoffstaette for the tip.

This fixes https://github.com/draios/sysdig/issues/895.
---
 userspace/libsinsp/marathon_http.cpp | 1 -
 userspace/libsinsp/mesos_http.cpp    | 1 -
 userspace/sysdig.project             | 1 -
 3 files changed, 3 deletions(-)

diff --git a/userspace/libsinsp/marathon_http.cpp b/userspace/libsinsp/marathon_http.cpp
index f7937b681..2e4449ad9 100644
--- a/userspace/libsinsp/marathon_http.cpp
+++ b/userspace/libsinsp/marathon_http.cpp
@@ -7,7 +7,6 @@
 #include "marathon_http.h"
 #include "curl/curl.h"
 #include "curl/easy.h"
-#include "curl/curlbuild.h"
 #define BUFFERSIZE 512 // b64 needs this macro
 #include "b64/encode.h"
 #include "sinsp.h"
diff --git a/userspace/libsinsp/mesos_http.cpp b/userspace/libsinsp/mesos_http.cpp
index 18ac9f5b5..c6bfb5810 100644
--- a/userspace/libsinsp/mesos_http.cpp
+++ b/userspace/libsinsp/mesos_http.cpp
@@ -7,7 +7,6 @@
 #include "mesos_http.h"
 #include "curl/curl.h"
 #include "curl/easy.h"
-#include "curl/curlbuild.h"
 #include "sinsp.h"
 #include "sinsp_int.h"
 #include "sinsp_curl.h"
diff --git a/userspace/sysdig.project b/userspace/sysdig.project
index 5d00f18f2..76a18f0a8 100644
--- a/userspace/sysdig.project
+++ b/userspace/sysdig.project
@@ -752,7 +752,6 @@
             </VirtualDirectory>
             <VirtualDirectory Name="include">
               <VirtualDirectory Name="curl">
-                <File Name="../build/release/curl-prefix/src/curl/include/curl/curlbuild.h"/>
                 <File Name="../build/release/curl-prefix/src/curl/include/curl/easy.h"/>
                 <File Name="../build/release/curl-prefix/src/curl/include/curl/stdcheaders.h"/>
                 <File Name="../build/release/curl-prefix/src/curl/include/curl/curlrules.h"/>
